### PR TITLE
feat: add support for adding multiple files to a pack at once

### PR DESCRIPTION
**Pull request by ozxybox**
This allows users to add multiple files using the "Add File" context menu rather than needing to always use add folder

### CODE REVIEW

File: src/gui/Window.cpp [hunk 1]
```diff
@@ -873,6 +873,14 @@ void Window::addFile(bool showOptions, const QString& startDir, const QString& f
 	this->markModified(true);
 }
 
+
+void Window::addFiles(bool showOptions, const QString &startDir) {
+	// Add multiple files using the multiple file selector
+	QStringList files = QFileDialog::getOpenFileNames(this, tr( "Open Files" ));
```
Comment (craftablescience) [inline src/gui/Window.cpp:879]: extra space in the tr :P

speaking of tr, should the translations in `src/gui/EntryContextMenuData.h` be updated to say `Add Files` plural?

oh, and should `Edit > Add File...` be updated to use this method as well?

File: src/gui/Window.cpp [hunk 2]
```diff
@@ -873,6 +873,14 @@ void Window::addFile(bool showOptions, const QString& startDir, const QString& f
...
+	for (const QString& path : files)
```
Comment (craftablescience) [inline src/gui/Window.cpp:880]: nit: should use braces for single lines

File: src/gui/Window.cpp (hunk 1 above)
Comment (ozxybox) [reply src/gui/Window.cpp:879]: This PR does update `Edit > Add File...`, it's higher up in the same file.

Will fix the space and text!

File: src/gui/Window.cpp (hunk 2 above)
Comment (ozxybox) [reply src/gui/Window.cpp]: fixed!

File: src/gui/Window.cpp (hunk 1 above)
Comment (ozxybox) [reply src/gui/Window.cpp]: fixed! also updated the other instances to be plural

Comment (craftablescience) [reply src/gui/Window.cpp]: > This PR does update `Edit > Add File...`, it's higher up in the same file.

im tired

**Review by craftablescience — APPROVED** [review]

File: src/gui/Window.cpp (hunk 1 above)
Comment (ozxybox) [reply src/gui/Window.cpp]: No worries!

**Pull request merged**